src/utils/filter-films.js

```diff
@@ -0,0 +1,2 @@
+export const filterFilms = (films, filters) =>
+  films.filter((film) => !filters.some(({ type, value }) => (film[type] !== value)));
```

Comment:
two filters have another logic . They don't work in this function.
For examle filter by time. It should filter values that greater that picked time .
may be I can replace filter logic to FILTER in constants ?